trace.c

```diff
@@ -61,7 +65,7 @@ long int RETRACE_IMPLEMENTATION(ptrace)(enum __ptrace_request request, ...)
 		case PT_READ_D:
                         request_str = "PT_READ_D";
                         break;
-		case PT_READ_U:
+		case 3: /* PT_READ_U: */
```

Comment:
I agree with @efiop , #ifdndef this out for FreeBSD, and add a `default:` to catch